core/src/main/scala/io/finch/Endpoint.scala

```diff
@@ -494,4 +490,24 @@ object Endpoint {
    * Generically derive a very basic instance of [[Endpoint]] for a given type `A`.
    */
   def derive[A]: GenericDerivation[A] = new GenericDerivation[A]
+
+  implicit def applicativeEndpoint: Applicative[Endpoint] = new Applicative[Endpoint] {
+
+    override def pure[A](x: A): Endpoint[A] = new Endpoint[A] {
+      override def apply(input: Input): Result[A] = None
```

Comment:
Let's do `Some()` here with `Eval.now`. We already have a value `x`, why not to put it into an `Endpoint` context?